### ISSUE TITLE
Copy/paste not working with CtD - Internal Server Error

### ISSUE THREAD

**Issue by kanarinka** [mitmedialab/DataBasic#307]
For example when I copy paste from libre office this:

source	target
a	b
b	c
c	d
d	b
f	e
e	f
e	p
a	e

or this, made with tabs:
source    target 
Alice    Bob 
Rahul    Catherine 
Catherine    Alice 
Bob    Rodrigo 
Orlando    Adiya 
Adiya    Catherine 
Rahul    Gus 
Cosmo    Bob

**Comment by kanarinka** [comment]
@rahulbot - I encountered this in my workshop setting! Both with pasted and uploaded data. Here's an example: https://docs.google.com/spreadsheets/d/18epHDk-rmntHO4dN-lFZZgceD0g8mVPJ1__ncHvcHgo/edit#gid=0 

Got internal server errors on copy/paste and upload for that data. Once I copied that data from that google doc into our CtD template it actually did work. 

Let's prioritize fixing this bc it definitely impacting my ability to run the workshop with CtD.
